### ISSUE TITLE
Is that a new behavior to Phalcon\Tag::prependTitle or it is a bug?

### ISSUE THREAD

**Issue by borisdelev** [phalcon/cphalcon#12233]
> **In Phalcon v2.x when i use tag:prependTitle the title is prepended**

```php
Phalcon\Tag::setTitle("PAGE");
Phalcon\Tag::prependTitle("test1");
Phalcon\Tag::prependTitle("test2");

// Title now is "test2 - test1 - PAGE" (depends of separator)
```
> **In Phalcon v3.x when i use tag:prependTitle the title replaced**

```php
Phalcon\Tag::setTitle("PAGE");
Phalcon\Tag::prependTitle("test1");
Phalcon\Tag::prependTitle("test2");

// Title now is "test2 - PAGE" (depends of separator)
```
### So which behavior is right?
Sorry for my English.
Thanks!

### Details

* Phalcon version: 3.x
* PHP Version: 7
* Operating System: Windows 10
* Server: Apache

**Comment by butamuh4o** [comment]
Yep, sure. :) thanks

**Comment by butamuh4o** [comment]
Finally. I did it :) Good luck all, and thanks for your patience.

**Comment by sergeyklay** [comment]
Fixed in the `3.0.x` branch.
